class.ext_update.php

```diff
@@ -121,7 +132,7 @@ protected function oldFormatClasses() {
         $result = $GLOBALS['TYPO3_DB']->exec_SELECTquery(
             'tx_dlf_formats.uid AS uid,tx_dlf_formats.type AS type',
             'tx_dlf_formats',
-            'tx_dlf_formats.class NOT LIKE "%\\\\\\\\%"' // We are looking for a single backslash...
+            'tx_dlf_formats.class IS NOT NULL AND tx_dlf_formats.class != "" AND tx_dlf_formats.class NOT LIKE "%\\\\\\\\%"' // We are looking for a single backslash...
```

Comment:
Another reason to not set `class = none` for IIIF in `ConfigurationForm.php`: It would always trigger this update.

Reply:
True. This change however is still relevant as `class` may now be empty.